doc_src/string.txt

```diff
@@ -106,6 +106,7 @@ The following subcommands are available:
 <bs>c</bs>g
 \endfish
 
+<<<<<<< c1ddfa5c636b8b353e56e10b66601ad1751cafcc
```

Comment:
These are conflict markers that need to be removed (the "====" in line 124 and ">>>>" in 182 as well).